mailpoet/lib/Form/Listing/FormListingRepository.php

```
@@ -61,7 +61,9 @@ protected function applySorting(QueryBuilder $queryBuilder, string $sortBy, stri
   }
 
   protected function applySearch(QueryBuilder $queryBuilder, string $search, array $parameters = []) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parameters'. (undefined)

(UnusedFormalParameter)